docs/release-notes/release-notes-0.13.1.md

```diff
@@ -0,0 +1,30 @@
+# Release Notes
```

Comment:
As I haven't really worked with the `lnd` release note flow yet, I'd just like to post a general question about the flow here so I understand the normal workflow during an actual release:

In the release PR, we'll remove all headers here which have no entry, is that correct?
Do we also add a new file for the next release in the release PR, or is that done in the first PR that gets merged?

Reply:
> In the release PR, we'll remove all headers here which have no entry, is that correct?

yeah we can just remove what we dont need. Just added these for some structure. Not strict at all

Reply:
> Do we also add a new file for the next release in the release PR, or is that done in the first PR that gets merged?

Again not strict. I'd lean towards the later though

Reply:
Ok cool, thanks for the clarification!